### DISCUSSION TITLE
通过Actions打包时，一直无法打包APP，里面有报错

### DISCUSSION THREAD

**Discussion by keventseng** [tw93/Pake#278] [General]
<img width="1062" alt="image" src="https://user-images.githubusercontent.com/5266604/215256713-cd511c4c-e587-4703-aa41-2b0f9496faa4.png">

**Comment by tw93** [comment]
试试同步一下最新的master，此外你有删除一些图片文件吗？还是可以不删除

**Reply by Tlntin** [reply]
看他仓库记录应该是解决了。

**Reply by keventseng** [reply]
解决了，感谢！
